commercetools-sdk-base/src/main/java/io/sphere/sdk/search/model/FilteredFacetExpressionImpl.java

```diff
@@ -25,4 +25,9 @@ public boolean equals(Object o) {
     public String alias() {
         return super.alias();
     }
+
+    @Override
```

Comment:
Yep, that's precisely the reason why I left open the option of the mistake, because it's an impl class and that doesn't sound correct. But maybe it's a special case, I can't remember. That's why better let's test it.